src/client.js

```diff
@@ -567,6 +567,74 @@ class Client extends EventEmitter {
         });
     }
 
+    createChannel(channelData, teamID, callback) {
+        const postData = {
+            team_id: teamID,
+            name: channelData.name,
+            display_name: channelData.display_name,
+            type: channelData.type,
+        };
+        if (channelData.purpose) {
+            postData.purpose = channelData.purpose;
+        }
+        if (channelData.header) {
+            postData.header = channelData.header;
+        }
+
+        return this._apiCall('POST', '/channels', postData, (data, _headers) => {
+            this.logger.info('Created Channel.');
+            if (callback != null) { return callback(data); }
+            return null;
+        });
+    }
+
+    getChannelByName(channelName, teamID, callback) {
```

Comment:
This API get channel for a teamID.

`getTeamChannelByName(teamID, channelName, callback)`